smrf/data/loadGrid.py

```diff
@@ -263,7 +263,10 @@ def load_from_netcdf(self):
         # GET THE TIMES
         t = f.variables['time']
         time = nc.num2date(t[:].astype(int), t.getncattr('units'), t.getncattr('calendar'))
-        time = [tm.replace(microsecond=0) for tm in time] # drop the milliseconds
+        # Drop milliseconds and prepare to use as pandas DataFrame index
+        time = pd.DatetimeIndex(
+            [str(tm.replace(microsecond=0)) for tm in time]
+        )
```

Comment:
The main change here

Reply:
Hmmm, interesting. Better to have it more explicitly defined as a `DateTimeIndex` anyways.